<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contactCreator</class>
 <widget class="QDialog" name="contactCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>433</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>653</width>
    <height>433</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>653</width>
    <height>433</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>创建新的联系人</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*为必须填写的项目</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="savet_Button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>390</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>备注</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="note_textedit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>531</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>581</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>姓名*</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_edit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>电话</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tel_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>EMail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mail_edit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>QQ/Wechat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="qq_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>生日</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gender_combo">
        <item>
         <property name="text">
          <string>男</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>女</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>跨性别</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>分类</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sort_combo">
        <item>
         <property name="text">
          <string>双亲</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>兄弟姐妹</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>亲戚</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>爱人</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>朋友</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>同事</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>同学</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>老师</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>上司</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>下属</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="add_edit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
